extensions/amp-story/1.0/pagination-buttons.js

```diff
@@ -244,28 +229,25 @@ export class PaginationButtons {
     }
   }
 
+  /**
+   * Updates element based on visibility state.
+   * @param {?PaginationButton} paginationButton
+   * @param {boolean} isVisible
+   * @private
+   */
+  toggleEnabled_(paginationButton, isVisible) {
+    const {buttonElement, element} = paginationButton;
+    element?.classList.toggle('i-amphtml-story-button-hidden', !isVisible);
+    buttonElement?.toggleAttribute('disabled', !isVisible);
```

Comment:
It does since the element is still visible, it just has a lowered opacity (I updated the naming). Button elements will receive focus unless `disabled` is explicitly set on them.